doc/ElementsApplicationPattern/Functions/ConceptOfConfigFile/ConceptOfConfigFile.md

```diff
@@ -1,39 +1,42 @@
-For example, a non-backward compatible modification to a function (e.g., a bug fix) would require changing the version number in the URI for calling this function.  
-If address information were hardcoded into the client, the client's program code would require an update in such cases.  
+For example, a non-backward compatible modification to a function (e.g. a bug fix) would require changing the version number in the URI for calling this function. If address information were hardcoded into the client, the client's program code would require an update in such cases.  
 Consequently, a change to an application would require changes to other applications.  
 Such propagating need for adaptation would make simple adjustments very cumbersome; and the architecture would be rigid against further development.  
 Therefore, address information needs to be configurable in some way and it needs to be saved independently from the program code.  
 This is the main purpose of the CONFIGfile.  
 
+**Content: address information and parameter values**  
 The CONFIGfile does not just store address information, but also all kinds of parameter values.  
 Parameters might be required for e.g., formulas, algorithms, or processes that are executed inside the application.  
 Such parameters are usually of primitive data types like Integer or String.  
 
-Some applications build their own data during runtime.  
-For example, the ExecutionAndTraceLog is creating a record for every request executed in the application layer.  
-Such data would be called ApplicationData.  
-The CONFIGfile shall not contain ApplicationData, but it might contain information that is required for connecting a separate file or database that is holding the ApplicationData.  
+**Not part of the content: Application Data**
+- Some applications build their own data during runtime. Such data would be called ApplicationData.
+- Example: the ExecutionAndTraceLog is creating a record for every request executed in the application layer..
+- The CONFIGfile shall not contain ApplicationData, but it might contain information that is required for connecting a separate file or database that is holding the ApplicationData.  
 
-Immediately after launching, the application is reading the CONFIGfile.  
+### CONFIGfile
+Immediately after launching, the application is reading the CONFIGfile.
 Consequently, the values that are defined in the CONFIGfile represent the initial state of the application.  
 
-The CONFIGfile is written in JSON format to support being read by JavaScript based applications.  
-It could be manually altered if required but operating the updated configuration would require restarting the application.  
-The applications, which are based on the MW SDN ApplicationPattern, are supporting changing their configuration via the REST API during runtime.  
+_ConfigFile changes_  
+The CONFIGfile is written in JSON format to support being read by JavaScript based applications.
```

Comment:
To my knowledge, JSON data could be handled in any programming language other than JS like Java, python , php etc.. i think we should reconsider this statement

Reply:
I think Thorsten especially mentioned JavaScript here, because this is what's being used in the project. I added a note that other languages can handle it too.)